test/rbenv.bats

```diff
@@ -149,39 +154,61 @@ OUT
 
 @test "not enough arguments for rbenv-install" {
   stub_ruby_build
+  stub rbenv-help 'install : true'
+
   run rbenv-install
   assert_failure
-  assert_output_contains 'Usage: rbenv install'
+  unstub rbenv-help
 }
 
 @test "too many arguments for rbenv-install" {
   stub_ruby_build
+  stub rbenv-help 'install : true'
+
   run rbenv-install 2.1.1 2.1.2
   assert_failure
-  assert_output_contains 'Usage: rbenv install'
+  unstub rbenv-help
 }
 
 @test "show help for rbenv-install" {
   stub_ruby_build
+  stub rbenv-help 'install : true'
+
   run rbenv-install -h
   assert_success
+  unstub rbenv-help
+}
+
+@test "rbenv-install has usage help preface" {
+  run extract_usage_from rbenv-install
```

Comment:
How about instead of having `extract_usage_from` function, simply do

```sh
run head -3 `which rbenv-install`
```

It seems that it could pass the test.